apps/gallery/js/ImageEditor.js

```diff
@@ -13,16 +13,20 @@ var editOptionButtons =
 var editBgImageButtons =
   Array.slice($('edit-options').querySelectorAll('a.bgimage.button'), 0);
 
+var editAspectMoreButtons =
+   Array.slice($('aspect-ratio-options-view').querySelectorAll('button'), 0);
+
 // Edit mode event handlers
 $('edit-exposure-button').onclick = setEditTool.bind(null, 'exposure');
 $('edit-crop-button').onclick = setEditTool.bind(null, 'crop');
 $('edit-effect-button').onclick = setEditTool.bind(null, 'effect');
 $('edit-enhance-button').onclick = setEditTool.bind(null, 'enhance');
-$('edit-crop-none').onclick = undoCropHandler;
 $('edit-header').addEventListener('action', onCancelButton);
 $('edit-tool-apply-button').onclick = applyEditTool;
 $('edit-save-button').onclick = saveEditedImage;
+
 editOptionButtons.forEach(function(b) { b.onclick = editOptionsHandler; });
+editAspectMoreButtons.forEach(function(b) {b.onclick = editAspectMoreHandler;});
```

Comment:
You could also register a single handler on the menu or form element and let the click bubble up, then use e.target instead of this in the handler function. But I guess doing it this way more closely parallels the existing code, so is probably better in this case.